src/api/handlers/period.js

```diff
@@ -114,7 +117,10 @@ module.exports.delete = {
         return PeriodResources
             .delete(per_id, userId)
             .then(
-                () => reply().code(204)
```

Comment:
nope, the call should be idempotent. Think it that way. There is a period. You send delete on it. It will return 204. You call delete again, the outcome is the same, there is no period there afterwards, therefore it should be 204